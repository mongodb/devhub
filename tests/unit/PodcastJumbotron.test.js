import React from 'react';
import { shallow } from 'enzyme';

import mockData from './data/PodcastJumbotron.test.json';
import PodcastJumbotron from '~components/dev-hub/podcast-jumbotron';

it('renders correctly', () => {
    const tree = shallow(<PodcastJumbotron {...mockData} />);
    expect(tree).toMatchSnapshot();
});
