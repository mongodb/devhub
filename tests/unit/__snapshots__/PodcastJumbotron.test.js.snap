// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<HeroBanner
  background="test-url"
  breadcrumb={
    Array [
      Object {
        "label": "Home",
        "target": "/",
      },
    ]
  }
>
  <Title
    collapse={true}
  >
    Test Title
  </Title>
  <PostMetaLine>
    <DateTextContainer>
      <DateText
        collapse={true}
      >
        Published: 
        21 Apr 2021
      </DateText>
    </DateTextContainer>
    <BlogTagList />
  </PostMetaLine>
  <BylineBlock />
</HeroBanner>
`;
